RELEASE.md

```diff
@@ -1,3 +1,34 @@
+# Release 2.2.1
+
+This patch release includes a rollup to HEAD so as to support required features
+for uploading graphs to TensorBoard.dev.
+
+## TensorBoard.dev updates
+
+- TensorBoard.dev now renders model graphs, for experiments uploaded starting
+  from this release.
+
+## Features
+
+- TensorBoard.dev graph rendering support
```

Comment:
Shouldn't anything related to TensorBoard.dev go in the TensorBoard.dev updates section?  That would make more sense IMO than separating out the server-side changes from the CLI changes.

In the past we've been kind of indiscriminate about this and put CLI changes either under "TensorBoard.dev updates" (2.1.0) or "Features" (2.2.0), but in a case where we have both server-side and CLI changes I would definitely expect them to be grouped together, and IMO better easier to be consistent overall if we always separate out the uploader changes in their own section.